### ISSUE TITLE
FR: Command line options

### ISSUE THREAD

**Issue by Knud3** [kwizatz88/GsyncSwitch#12]
Add command line options like `GsyncSwitch -a AllowVRR=1` and `GsyncSwitch -a AllowVRR=0` for turning G-Sync on and off programmed. Then there is no need to keep GsyncSwitch running for tray icon.

**Comment by e270889o** [comment]
> 
You can use an old version. That version has a GSyncSwitchEXE.exe file that you can call from CMD. It only switches Gsync (you dont know the actual status, just switch to the other). I use it with sunshine
GsyncSwitch_V2.0
released this Jun 7, 2022

But yes, it could be great that we have the option of using command line options in the newer release

**Comment by CorpusBava** [comment]
I'd like to see the ability to have Gsyncswitch toggle gsync off then on again with just a single click, perhaps using a batch file or even an autohotkey script.  My LG C2 OLED TV is very buggy when my Windows box comes out of sleep on my RTX 3070 video (I get pixel flashing of the screen every few seconds similar to the VRR refresh rate bug).  The only thing that fixes it is disabling gsync then turning it back on.  as it stands, I have to click gsyncswitch once to disable gsync then again to re-enable it.  Would like to do that in one-click.  Any suggestions?

**Comment by kwizatz88** [comment]
What mouse do you have ?
You might be able to assign a macro to a button, like gsyncswitchExe call, pause 5s, gsyncSwitchExe call

You can also use a .cmd file where you put : 
GsyncSwitchEXE.exe
ping -n 6 127.0.0.1 > nul
GsyncSwitchEXE.exe

(2d line will pause 5s by making 6 pings to localhost)

**Comment by CorpusBava** [comment]
GsyncSwitchEXE is from the old v2.0 build of GsyncSwitch, correct?  Yes, I tried that, finding a very similar batch file on reddit but with only a 2 sec wait between switching.  However, I started experiencing a horrible bug with my LG C2 TV when I ran GsyncSwitchEXE where the screen would go blank and just the message "Invalid Format" would come up and I had to unplug my HDMI cable and re-plug it to get video back.  It actually prompted me to do an acronis restore of Windows it was so unnerving.  But I think i had something misconfigured or corrupted in my Windows install that wasn't GsyncSwitch's fault since I just tried it again and it worked normally.  I'll see if my system remains stable this time.

**Comment by Knud3** [comment]
LG C2 VRR problem is exact reason why I also used this. Scheduled for log on working fine. I am also discovering this project to help further: https://github.com/Maassoft/ColorControl

**Comment by CorpusBava** [comment]
Almost can't believe that I paid $1350 for my LG C2 "65 TV and it has such a fundamental hardware problem when connected to a PC with coming out of sleep.  If I had it to do again, I'd likely get a different brand of OLED TV.  Did you ever find a solution other than turning gsync off then off again?  Luckily, GsyncSwitch makes it easy with just a click or two instead of going into Nvidia CP but it's still a workaround that I shouldn't have to do if LG actually engineered their gsync/VRR modules properly..

**Comment by Knud3** [comment]
For me its have been sunny days after I changed from Samsung. This VRR problem is VERY minor problem. After three warranty exchange and waiting over half a year for service.

**Comment by CorpusBava** [comment]
Yeah, I heard Samsung TVs these days cause a world of grief for the consumer.  I should count my blessings that the worst thing I experience is pixel flashing when my PC comes out of sleep because of gsync gamma hardware issues with LG.  Poor widdle me and my first-world problems that I have to click on a batch file to toggle gsync on/off again as a workaround to fix it when others have dealt with multiple RMAs because of Samsung's poor customer service.  I cried because I had no properly working VRR/gsync on my new TV... until I saw a man who had no TV because of Samsung lol . 

Thanks, kwizatz88, by the way for posting that batch file and for making GsyncSwitch in the first place.  Just used it not 5 minutes ago after my PC woke and I got the pixel flashing.  Works a treat.

**Comment by Knud3** [comment]
Did not mean to make your problem any less real or frustrating. Nowadays "just working" products are next to no existing. You just have to pick your poison and hope for the best.

And sorry for off topics...

**Comment by ecast040** [comment]
Any plans to add a hotkey feature? would be nice to have this integrated with the latest build.
